Conceptpower+Spring/src/main/webapp/WEB-INF/views/home.jsp

```diff
@@ -191,21 +191,26 @@ function prepareMergeConcept(conceptId) {
 function createWrapper(wrapperId) {
   window.location = '${pageContext.servletContext.contextPath}/auth/conceptlist/addconceptwrapper?wrapperId=' + wrapperId;
 }
+
 $(document).ready(function(){
-    $(".fa-exclamation-triangle").click(function() {
+    $(document).on("click", ".fa-exclamation-triangle", function() {
```

Comment:
Since we are adding exclamation icons dynamically after submitting request, the event handlers will not be created corresponding to it. ().click() will be separate for each corresponding html element.

Having $(document).on("click") there will be only one handler for the entire page matching the corresponding html element. It will handle the dynamically created elements as well.